apps/datahub-e2e/src/e2e/organization-page.cy.ts

```diff
@@ -0,0 +1,125 @@
+import 'cypress-real-events'
+
+describe('organizations', () => {
+  beforeEach(() => {
+    cy.visit('organization/Barbie%20Inc.')
+
+    // aliases
+    cy.get('gn-ui-navigation-button').as('backButton')
+    cy.get('[data-cy="organizationHeaderName"]').as('organizationHeaderName')
+    cy.get('[data-cy="organizationHeaderWebsiteLink"]').as(
+      'organizationHeaderWebsiteLink'
+    )
+    cy.get('[data-cy="organizationDescription"]').as('organizationDescription')
+    cy.get('gn-ui-max-lines').contains('Read more').as('readMoreButton')
+    cy.get('[data-cy="organizationLogo"]').as('organizationLogo')
+    cy.get('[data-cy="organizationDatasetCount"]').as(
+      'organizationDatasetCount'
+    )
+    cy.get('[data-cy="organizationEmail"]').as('organizationEmail')
+    cy.get('[data-cy="organizationPageLastPublishedDatasets"]').as(
+      'organizationPageLastPublishedDatasets'
+    )
+    cy.get(
+      '[data-cy="organizationDetailsLastPublishedDatasetsSearchAllButton"]'
+    ).as('organizationDetailsLastPublishedDatasetsSearchAllButton')
```

Comment:
As mentioned in the according file, this data-cy var is way too long to be readable. The alias is also too long.

Reply:
It's explicit! No collision possible 😄